pkg/accountGRPC/account.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package accountGRPC;
+option go_package = "github.com/stasBigunenko/monorepa/pkg/account/proto";
+
+import "google/protobuf/empty.proto";
+
+service UserGRPCService {
+  rpc Get(User_id) returns (Accounts) {}
+  rpc Create(User_id) returns (Obj) {}
+  rpc Update(Balance) returns (Balance) {}
```

Comment:
It would be logically more clear if we update Account, not Balance

Reply:
UpdateAccount(Account) returns (Account)

Reply:
done